docs/DeveloperGuide.adoc

```diff
@@ -115,21 +130,23 @@ NOTE: The lifeline for `DeleteCommandParser` should end at the destroy marker (X
 === Model component
 
 .Structure of the Model Component
-image::ModelClassDiagram.png[]
+image::BetterModelClassDiagram.png[]
```

Comment:
There should not be floating models in the diagrams. need an arrow to bind it to something